types/src/app.d.ts

```diff
@@ -0,0 +1,20 @@
+// --- App ---
+// Any types which are shared between different parts of the App
+
+export type Breadcrumb = {
+  name: string;
+  href: string;
+};
+
+export type FilterItem = {
```

Comment:
Could you add some comments about what each option is? It's not clear to me just from reading this type definition

Reply:
I've added comments here ✊